companion/manifest.json

```
@@ -3,7 +3,7 @@
 	"name": "generic-osc",
 	"shortname": "osc",
 	"description": "Direct: OSC plugin for Companion",
-	"version": "0.0.0",
+	"version": "2.3.0",
```

This should stay at 0 and it gets set automagically from package.json:
```suggestion
	"version": "0.0.0",
```